test/evergreen.yml

```diff
@@ -4099,10 +4099,10 @@ buildvariants:
     - name: ".stress-test-ppc-1"
     - name: ".stress-test-ppc-2"
 
-- name: ubuntu1804-zseries
-  display_name: "~ Ubuntu 18.04 zSeries"
+- name: rhel80-zseries
+  display_name: "~ RHEL80 zSeries"
```

Comment:
Ditto, and a space before 8, i.e. `~RHEL 8 zSeries`

Reply:
The space is consistent with the other display names, are you sure to remove this one?